app/src/main/java/com/duckduckgo/app/cta/ui/CtaViewModel.kt

```diff
@@ -470,6 +502,16 @@ class CtaViewModel @Inject constructor(
 
     fun isSuggestedSiteOption(query: String): Boolean = onboardingStore.getSitesOptions().map { it.link }.contains(query)
 
+    fun getCohortOrigin(): String {
+        return when {
+            extendedOnboardingFeatureToggles.testPrivacyProOnboardingCopyNov24().isEnabled(Cohorts.PROTECTION) -> "_${Cohorts.PROTECTION.cohortName}"
```

Comment:
You can do `extendedOnboardingFeatureToggles.testPrivacyProOnboardingCopyNov24().getCohort()` directly.